app/layout/layout.phtml

```diff
@@ -77,10 +77,10 @@ if (_t('gen.dir') === 'rtl') {
 <?php
 	$msg = '';
 	$status = 'closed';
-	if (!empty($this->notification)) {
-		$msg = $this->notification['content'];
-		$status = $this->notification['type'];
-
+	$notif = Minz_Request::getNotification();
+	if (!empty($notif)) {
```

Comment:
```suggestion
	if ($notif !== null && array_key_exists('content', $notif) && array_key_exists('type', $notif)) {
```

Reply:
The check is already done by type declaration (this code already passes PHPStan Level 9). And if anything, `array_key_exists` does not seem like the correct test as it accepts `null`, which we do not want.
We could consider using `??` below for providing default values, but I do not think it is necessary for now.